istio/README.md

```diff
@@ -52,69 +48,80 @@ instances:
       - request_protocol
       - connection_security_policy
 ```
-   
-**Note**: The `connectionID` Prometheus label is excluded. The [sample istio.d/conf.yaml][8] also has a list of suggested labels to exclude.
+These excluded labels are set in the existing `istio.d/auto_conf.yaml` file.
 
+##### Control plane configuration
+To monitor the Istio control plane and report the `mixer`, `galley`, `pilot`, and `citadel` metrics, you must configure the Agent to monitor the `istiod` deployment. In Istio `v1.5+`, apply the following Autodiscovery Annotations as pod annotations for the deployment `istiod` in the `istio-system` namespace:
 
-   Istio mesh metrics are only available from `istio-proxy` containers which are supported out-of-the-box with Autodiscovery, see [`istio.d/auto_conf.yaml`][9].   
+```yaml
+ad.datadoghq.com/discovery.check_names: '["istio"]'
+ad.datadoghq.com/discovery.init_configs: '[{}]'
+ad.datadoghq.com/discovery.instances: |
+     [
+       {
+         "istiod_endpoint": "http://%%host%%:15014/metrics",
+         "use_openmetrics": "true"
```

Comment:
Can we include `exclude_labels` here as well?

Reply:
The existing `exclude_labels` that we reference are specific to the data plane portion of this integration. Those labels don't show up in the control plane istiod output so they shouldn't be necessary for this specific portion of the integration. 

But if there any specific `exclude_labels` that you wanted for these metrics I can get those added into the docs:
- https://github.com/DataDog/integrations-core/blob/master/istio/tests/fixtures/1.5/istiod.txt

Reply:
Ah that's true, my mistake.